content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -459,6 +459,35 @@ validation and auto-completion.
 ⚠️ Note that using the `checkpoint` field in `dvc.yaml` is not compatible with
 `dvc repro`.
 
+## Standalone plots configurations
+
+The list of plots contains one or more user-defined
+[plots](/doc/command-reference/plots#standalone-plots). Here's an example that
+tells DVC that `auc.json` is viable for visualization:
+
+```yaml
+stages:
+  build:
+    cmd: python train.py
+    deps:
+      - features.csv
+    outs:
+      - model.pt
+      - auc.json
+    metrics:
+      - accuracy.txt:
+          cache: false
+plots:
+  auc.json:
+    x: fpr
+    y: tpr
+```
+
+Note that in this example we don't have to specify `auc.json` as a plot output.
+Standalone plots accept any file from repository. Refer to the plots  
+[documentation](/doc/command-reference/plots#syntax) for more information on
+plots syntax.
```

Comment:
```suggestion
Note that we didn't have to specify `auc.json` as a plot in the stage. In fact,
Top-level plot definitions can use any file in the <abbr>workspace</abbr> or
<abbr>cache</abbr>.

[plots]: /doc/command-reference/plots#standalone-plots
```

Reply:
I am not sure we should mention cache - "Any file from repository" covers all files - tracked by git,dvc and even those that were just copied into the repo folder and not tracked by anything.

Reply:
Does this work with non-Git projects?

Reply:
Yes

Reply:
Updated suggestion ☝🏼

Reply:
Updated